RevoValidation/src/Validation.swift

```diff
@@ -15,6 +15,8 @@ public class Validation {
     var okText = ""
     var delegate:ValidationDelegate?
     
+    var shouldShowFirstOneFlag:Bool = false
```

Comment:
treu el Flag de darrere, amb el should ja sabem que es bool, per tant ja sabem que es flag